bin/behat-test.sh

```diff
@@ -8,4 +8,4 @@ set -ex
 
 export BEHAT_PARAMS='{"extensions" : {"Behat\\MinkExtension" : {"base_url" : "http://'$TERMINUS_ENV'-'$TERMINUS_SITE'.pantheonsite.io"} }}'
 
-./vendor/bin/behat
+./vendor/bin/behat $*
```

Comment:
I don't have my own opinion on the best way to do this. I went looking for one and found 

> Well, let's just say: You almost always want a quoted "$@"!

http://wiki.bash-hackers.org/scripting/posparams#mass_usage

Is `$*` preferable over `"$@"` for some reason in this case?